src/modules/ModuleFarming/store/farming.ts

```diff
@@ -6,6 +6,12 @@ export const useFarmingStore = defineStore('farming', () => {
   const searchQuery = ref('')
   const sorting = ref<Sorting>(Sorting.Hot)
 
+  function resetFilter() {
+    stakedOnly.value = false
+    searchQuery.value = ''
+    sorting.value = Sorting.Hot
+  }
```

Comment:
I would make a factory:

```ts
const stateFactory = () => ({ stakedOnly: false, searchQuery: '', sorting: Sorting.Hot })
```

And then:

```ts
const state = ref(stateFactory())

const reset = () => {
  state.value = stateFactory()
}

return { ...toRefs(state), reset }
```

Reply:
Done

Reply:
Done